index.rst

```diff
@@ -221,5 +246,128 @@ Banging on the side of the strut to excite some kind of bending mode was less su
+
+.. image:: ./_static/On_telescope_22Nov22.jpg
+
+
+	   
+Fourth test - 23-Nov-22
+=================================
+
+The tests were repeated on 23-Nov-22.  The wind velocity outside the dome as measured by the Gemini weather station was 3.6 m/s from 302 degrees.  The slit was only about 50% open to prevent the sun from falling on the telescope.  The telescope and dome were aligned to 180 degrees.  The figure below shows the setup: 
+
+		
+.. image:: ./_static/Dome_Setup_23Nov22.jpg
+
+**Setup on 23-Nov-22**
+
+
+These images show the anemometer readings, the accelerometer readings, and the mount encoder data as the fan speed is ramped up:
+
+
+Measurements with Fan Speed at 0.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112300018.png
+
+Measurements with Fan Speed at 10.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112300023.png
+
+Measurements with Fan Speed at 20.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112300028.png
+
+Measurements with Fan Speed at 30.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112300033.png
+
+Measurements with Fan Speed at 40.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112300038.png
+
+Measurements with Fan Speed at 50.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112300043.png
+
+
+Summary of measurements
+-----------------------------------------------
+
+   .. image:: ./_static/Anemom_vs_Fan_Speed_23Nov22.png
+	      
+   .. image:: ./_static/Joint_Ane_Accel_Mount_Tests_23Nov22.png
+
+This is a summary of the measurements taken.  The total accelerometer power, and the mount jitter all ramp up as the fan speed is increased. All of the anemometer standard deviations increase as the fan speed is ramped, indicating increased turbulence.  In addition, the Uz mean speed also increases.  The anemometer readings show much more scatter than expected.
+
+Fifth test - 28-Nov-22
+=================================
+
+The tests were repeated on 28-Nov-22.  The wind velocity outside the dome as measured by the Gemini weather station was 3.4 m/s from 325 degrees.  By positioning the telescope and dome to 230 degrees, I was able to open the slit somewhat further without the sun falling on the telescope.  The figure below shows the setup: 
+
+		
+.. image:: ./_static/Dome_Setup_28Nov22.jpg
+
+**Setup on 28-Nov-22**
+
+
+These images show the anemometer readings, the accelerometer readings, and the mount encoder data as the fan speed is ramped up:
+
+
+Measurements with Fan Speed at 0.0 Hz
```

Comment:
boatload of power at 70 Hz in AuxTel-Truss Opt... but then seems to die out? any idea what that is?